genrl/deep/common/__init__.py

```diff
@@ -0,0 +1,27 @@
+from genrl.deep.common.actor_critic import MlpActorCritic  # noqa
+from genrl.deep.common.base import BaseActorCritic  # noqa
+from genrl.deep.common.buffers import PrioritizedBuffer  # noqa
+from genrl.deep.common.buffers import PushReplayBuffer  # noqa
+from genrl.deep.common.buffers import ReplayBuffer  # noqa
+from genrl.deep.common.logger import CSVLogger  # noqa
+from genrl.deep.common.logger import HumanOutputFormat  # noqa
+from genrl.deep.common.logger import Logger  # noqa
+from genrl.deep.common.logger import TensorboardLogger  # noqa
+from genrl.deep.common.noise import ActionNoise  # noqa
+from genrl.deep.common.noise import NormalActionNoise  # noqa
+from genrl.deep.common.noise import OrnsteinUhlenbeckActionNoise  # noqa
+from genrl.deep.common.policies import BasePolicy, CNNPolicy, MlpPolicy  # noqa
+from genrl.deep.common.rollout_storage import RolloutBuffer  # noqa
+from genrl.deep.common.trainer import OffPolicyTrainer, OnPolicyTrainer, Trainer
+from genrl.deep.common.utils import cnn  # noqa
+from genrl.deep.common.utils import get_env_properties  # noqa
+from genrl.deep.common.utils import get_model  # noqa
+from genrl.deep.common.utils import init_weights  # noqa
+from genrl.deep.common.utils import mlp  # noqa
+from genrl.deep.common.utils import safe_mean  # noqa
+from genrl.deep.common.utils import set_seeds  # noqa
+from genrl.deep.common.values import CnnValue  # noqa
+from genrl.deep.common.values import MlpValue  # noqa
+from genrl.deep.common.values import _get_val_model  # noqa
+from genrl.deep.common.values import get_value_from_name  # noqa
+from genrl.deep.common.values import CnnValue, MlpValue  # noqa
```

Comment:
Delete this whole file. In fact all files under `genrl/deep` must be deleted. We've changed the structure of the repo so these might have come due to some improper merging.